corehq/apps/export/forms.py

```diff
@@ -646,7 +646,11 @@ def get_form_data_from_export_instance_filters(cls, export_instance_filters, dom
                 )
 
             emwf_utils_class = CaseListFilterUtils if export_type is CaseExportInstance else EmwfUtils
-            emwf_data = [emwf_utils_class(domain).id_to_choice_tuple(str(x)) for x in selected_items]
+            emwf_data = []
+            for x in selected_items:
```

Comment:
nit: `x` is not a good name

Reply:
absolutely. 
Thanks for the quick review. Updating.

Reply:
👍